pkg/server/middleware/auth.go

```diff
@@ -109,8 +112,8 @@ func (um *UserMiddleware) setUser(c echo.Context, userID int) {
 
 func (um *UserMiddleware) extractToken(c echo.Context) (string, error) {
 	authHeader := c.Request().Header.Get("Authorization")
-	if authHeader != "" {
-		return strings.TrimPrefix(authHeader, "Bearer "), nil
+	if authHeader != "" && strings.HasPrefix(authHeader, bearerHeader) {
```

Comment:
This should be a case insensitive check. The RFC for Bearer declares that it should be case sensitive, but unfortunately that's not the fact. So also check for `bearer`.